testcases/expand.go

```diff
@@ -124,6 +125,9 @@ func applyQueryTweaks(tc *comparer.TestCase, tweaks []*config.QueryTweak) *compa
 			resTC.Start = resTC.Start.Truncate(resTC.Resolution)
 			resTC.End = resTC.End.Truncate(resTC.Resolution)
 		}
+		if (config.Substitution{}) != t.ApplyQuerySubstitution {
```

Comment:
(If I didn't suggest changing this to a map already, I would suggest changing optional query tweaks to a pointer so you could do the simpler `if t.ApplyQuerySubstitution != nil {` here.)

Reply:
You did.  But thanks for the Go help!  I never know when things will be nil vs. an empty map, etc.

Reply:
Done!